src/category/category.service.ts

```diff
@@ -53,7 +51,7 @@ export class CategoryService {
       },
     });
 
-    return GetCategoriesMapper.toRespone(category);
+    return GetCategoryMapper.toRespone(category);
```

Comment:
Mapper cannot be dedicated for an action. GetCategory is an action. this means you're naming suggesting that it is a mapper for GetCategory action. 

What if I want to reuse this in createCategory?